synfig-core/po/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+file(STRINGS LINGUAS LANGUAGES)
```

Comment:
I think we can put all of this strings inside `if(GETTEXT_FOUND)` condition.
After that you can move all this code to the function.

Reply:
I think making a `/cmake` folder and moving this code to a function in a file over there could be a good option too.

Reply:
I support that.